package com.simplecity.amp_library.model;

import com.simplecity.amp_library.interfaces.FileType;

@SuppressWarnings({"java:S1104", "java:S1444", "java:S131", "java:S1301", "java:S3776", "java:S3740", "java:S1066", "java:S1192", "java:S125", "java:S1118", "java:S117", "java:S1135", "java:S100", "java:S116"})
public class FolderObject extends BaseFileObject {

    @SuppressWarnings("java:S1104")

    public int fileCount;
    @SuppressWarnings("java:S1104")
    public int folderCount;

    public FolderObject() {
        this.fileType = FileType.FOLDER;
    }

    @Override
    public String toString() {
        return "FolderObject{" +
                "fileCount=" + fileCount +
                ", folderCount=" + folderCount +
                "} " + super.toString();
    }
}
